cli/examples/all.json

```diff
@@ -0,0 +1,47 @@
+[
+    1,
+    "All",
```

Comment:
Multiple occurrences:  In this and later examples the RuleSet name is same as the Rule name.  I think it would be clearer if you did this more like what you did with additional signer: "AdditionalSignerRuleSet" for the name.